src/lib.rs

```diff
@@ -26,10 +26,57 @@
 //! # #[cfg(not(all(feature = "rustls-native-certs", feature = "tokio-runtime", feature = "http1")))]
 //! # fn main() {}
 //! ```
+//!
+//! ## Example server
+//!
+//! ```no_run
+//! # #[cfg(all(feature = "rustls-native-certs", feature = "tokio-runtime", feature = "http1", feature = "acceptor"))]
+//! # fn main() {
+//! use hyper::server::conn::AddrIncoming;
+//! use hyper::service::{make_service_fn, service_fn};
+//! use hyper::{Body, Method, Request, Response, Server, StatusCode};
+//! use hyper_rustls::TlsAcceptor;
+//! use std::io;
+//! use std::fs::File;
+//!
+//! let mut rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
Would probably be good to add some an empty line before each comment to make this a bit easier to consume.